src/tic.h

```diff
@@ -299,6 +299,7 @@ typedef struct
     struct
     {
         u8 music_loop:1;
+        u8 music_sustain:1;
         u8 music_state:2; // enum tic_music_state
         u8 unknown:5;
     } flag;
```

Comment:
Please, move `u8 music_sustain:1;` after `u8 music_state:2;` because it breaks compatibility.
``` c
    struct
    {
        u8 music_loop:1;
        u8 music_state:2; // enum tic_music_state
        u8 music_sustain:1;
        u8 unknown:4;
    } flag;
```

Reply:
Also, pls use `u8 unknown:4;` because you don't fit 1 byte :)

Reply:
I added a static assertion here 9dace07 to check `tic_sound_state` size

Reply:
Okay, done :)